packages/optimizer/lib/transformers/RewriteAmpUrls.js

```diff
@@ -60,6 +60,7 @@ const {calculateHost} = require('../RuntimeHostHelper');
 class RewriteAmpUrls {
   constructor(config) {
     this.esmModulesEnabled = config.esmModulesEnabled !== false;
+    this.preloadEnabled = config.preloadEnabled === true;
```

Comment:
Should this only be honored if the boilerplate was removed?

Reply:
Yes, only if boilertplate has been removed. I'm wondering whether we should make it configurable at all. I'd tend to remove the option.

Reply:
Done in 17cd9bd